install/venvinstall.sh

```diff
@@ -8,14 +8,24 @@ if [ ! -d install ]; then
     exit 1
 fi
 
-# Create virtual environment
-python3 -m venv venv
-
-# Activate virtual environment
-. ./venv/bin/activate
+if [ ! -d venv ]; then
+    # Create virtual environment
+    echo "Creating python venv"
+    python3 -m venv venv
+fi
 
 # Upgrade pip
-pip install pip --upgrade
+./venv/bin/pip install pip --upgrade
 
 # Install requirements.txt
-pip install -r requirements.txt
+./venv/bin/pip install -r requirements.txt
+
+if [ $CIRCLECI ]; then
```

Comment:
should be `$CI` to make it more portable

Reply:
This is now fixed.